src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1471,25 +1472,33 @@ static void ConsoleCommandReplayStart(InteractiveConsole& console, const argumen
     std::string name = argv[0];
 
     auto* replayManager = OpenRCT2::GetContext()->GetReplayManager();
-    if (replayManager->StartPlayback(name))
+
+    try
     {
-        OpenRCT2::ReplayRecordInfo info;
-        replayManager->GetCurrentReplayInfo(info);
+        replayManager->StartPlayback(name);
+    }
+    catch (std::invalid_argument& e)
```

Comment:
```suggestion
    catch (const std::invalid_argument& e)
```
in other places as well